### PR TITLE
Feature/project detailst component(frontend)

### PR DESCRIPTION
**Pull request by MaksymFilypchuk**
project (project-routing.module, project.module, project.component, project-setting.component, project-create.component)

### CODE REVIEW

File: frontend/src/app/modules/project/project-settings/project-settings.component.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    this.getProject(this.projectId);
+  }
+  getProject(projectId: number) {
+      this.isLoading = true;
+      this.projectService
+      .getProjectById(projectId)
+        .subscribe(
+          (resp) => {
+            this.isLoading = false;
+            this.tempProjectName = resp.body.name;
```
Comment (VoBilyk) [inline frontend/src/app/modules/project/project-settings/project-settings.component.ts:39]: You don't have to return full response to a component. Let's return only body.

File: frontend/src/app/modules/project/project.component.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 28 28">
+          <path
+            d="M14 0C6.27 0 0 6.43 0 14.36c0 6.34 4.01 11.72 9.57 13.62.7.13.96-.31.96-.69 0-.34-.01-1.24-.02-2.44-3.89.87-4.72-1.92-4.72-1.92-.64-1.66-1.55-2.1-1.55-2.1-1.27-.89.1-.87.1-.87 1.4.1 2.14 1.48 2.14 1.48 1.25 2.19 3.28 1.56 4.07 1.19.13-.93.49-1.56.89-1.92-3.11-.36-6.38-1.59-6.38-7.09 0-1.57.55-2.85 1.44-3.85-.14-.36-.62-1.82.14-3.8 0 0 1.18-.39 3.85 1.47a12.8 12.8 0 013.5-.48c1.19.01 2.39.16 3.5.48 2.67-1.86 3.85-1.47 3.85-1.47.76 1.98.28 3.44.14 3.8.9 1 1.44 2.28 1.44 3.85 0 5.51-3.27 6.73-6.39 7.08.5.44.95 1.32.95 2.66 0 1.92-.02 3.47-.02 3.94 0 .38.25.83.96.69C23.99 26.07 28 20.7 28 14.36 28 6.43 21.73 0 14 0z"
+            fill-rule="evenodd" clip-rule="evenodd"></path>
+        </svg>
+      </a>
+    </div>
+    <div class="repo-badge">
+      <a title="Latest push build on default branch: ">
+        <img src="https://travis-ci.com/Maksym-Filypchuk/Snake.svg?branch=master&amp;status=unknown" alt="build:">
```
Comment (VoBilyk) [inline frontend/src/app/modules/project/project.component.html:30]: Could you move all the needed images to our project under the assets folder?

File: frontend/src/app/modules/project/project.component.ts [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  isLoading = false;
+  constructor(
+    private projectService: ProjectService,
+    private toastrService: ToastrNotificationsService,
+    private route: ActivatedRoute
+    )
+  {
+    this.route.paramMap.pipe(
+    switchMap(params => params.getAll('projectId')))
+    .subscribe(data => this.id = +data);
```
Comment (VoBilyk) [inline frontend/src/app/modules/project/project.component.ts:25]: If you want to convert string to number use Number(data) object better

File: frontend/src/app/modules/project/project-create/project-create.component.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  <h2 class="small-title">
+    Create Project
+  </h2>
+  <div class="form-croup-project">
+    <label for="name-input">
+      <span class="form-control-project">Project name</span>
+      <input type="text" [(ngModel)]="newProject.name" name="name" required autocomplete="off" #name="ngModel"
+        id="name-input" class="form-control" placeholder="My first project">
+    </label>
+    <div class="error" [hidden]="name.errors?.required == null || name.untouched">
```
Comment (VoBilyk) [inline frontend/src/app/modules/project/project-create/project-create.component.html:11]: Don't need strick checking. Please, rewrite all similar to: !name.errors?.required || name.untouched

File: frontend/src/app/modules/project/project-settings/project-settings.component.ts (hunk 1 above)
Comment (MaksymFilypchuk) [reply frontend/src/app/modules/project/project-settings/project-settings.component.ts:39]: its for 'put' and 'post' methods too, or only for 'get'?

Comment (VoBilyk) [reply frontend/src/app/modules/project/project-settings/project-settings.component.ts]: For all methods

**Pull request merged**